language-support/java/codegen/BUILD.bazel

```diff
@@ -216,6 +216,7 @@ daml_build_test(
         ":pkg1.dar": "pkg1.dar",
         ":pkg2.dar": "pkg2.dar",
     },
+    enable_scenarios = True,
```

Comment:
haven’t checked in detail but maybe we can just not have scenarios in here.

Reply:
yeah, this one just has 
```
scenario = do
  p <- getParty "p"
  submit p $ create (V1.A.T1 p)
  submit p $ create (V2.A.T2 p)
```
I think that should work as a `Script`, right? I'll do it in a separate PR though

Reply:
You can do it as a script but I’m not sure why we need it at all. I think we can just delete it and the tests should still work. But definitely  happy to do it in a separate PR.